packages/application-shell/src/components/authenticated/am-i-logged-in.tsx

```diff
@@ -24,7 +24,10 @@ const AmILoggedIn = (props: Props) => {
     TAmILoggedInQuery,
     TAmILoggedInQueryVariables
   >(AmILoggedInQuery, {
-    variables: { target: GRAPHQL_TARGETS.MERCHANT_CENTER_BACKEND },
+    variables: {
+      target: GRAPHQL_TARGETS.MERCHANT_CENTER_BACKEND,
+      skipTokenRetry: true,
```

Comment:
FYI: in #1677 we plan to remove all these **request context** variables from the query `variables` and pass them using `context`.

See https://merchant-center-application-kit-git-nm-migrate-apollo-client.commercetools.vercel.app/custom-applications/releases/2020-09-25-migrating-to-the-new-apollo-client-30

Should we maybe do that for this new flag `skipTokenRetry`?

Reply:
Sure. To make sure:

```js
const query = useQuery(Foo, {
   context: {
      skipTokenRetry: true
   }
})
```

right?

Reply:
Correct. Check the changes in the other PR if you want to be sure.
I can also help out if you want to.

Reply:
All good. Just wanted to make sure I got the final API right.

Reply:
Like this? :) a4ab29c

Reply:
yeah, that looks right to me